docs/analysis.rst

```diff
@@ -10,25 +10,143 @@ the final free energy output that you get. In theory, we could just print out
+In :ref:`visual <visual_analyze>` mode, a heat map shows the same data where darker shading indicates a denser exchange
+of states. Note that if a state does not exchange replicas, it is counted as "exchanging with itself." You want a
+state to exchange with at least one other state.
 
-.. todo:: Fill in analysis instructions and description.
+The Peron eigenvalue is computed as a quantitative metric of how well mixed the replicas are. This is computed as the
+:ref:`second eigenvalue of the transition state matrix <sim_hp_report>`. This quantity is the provides the estimate for
+how decomposable the transition state matrix is and how many iterations it would take for one state to swap to all
+replicas. The lower this number is, the better mixing has occurred.
 
-Analysis tools
+The :ref:`visual analyze<visual_analyze>` mode provides an additional qualitative guide for how individual replicas
+sampled each state. The state each replica is in is plotted as a function of time to see if any particular replicas
+are getting stuck in a single state.
+
+
+Free Energy Estimation
+^^^^^^^^^^^^^^^^^^^^^^
+
+The free energy difference and its error are estimated through the Multistate Bennet Acceptance Ratio implemented in
+PyMBAR. The equilibrated and decorrelated data :ref:`are analyzed by MBAR <mbar_algorithm>` with default values for the
+initial guess. If there are unsampled states, such as when
+:ref:`anisotropic-dispersion corrections are accounted for <ansiotropic_algorithm>`, then the free energy difference to
+those states are also estimated. The final :doc:`free energy difference of scientific interest <theory>`, such as
+binding or hydration, is taken when multiple phases' free energies are added together, along with any
+:ref:`standard state corrections <standard_state_algorithm>`.
+
+
+.. _analyze_usage:
+
+Analysis Usage
 --------------
-.. autosummary:
-    :toctree: api/generated/
 
-    analyze
+.. _auto_analyze:
+
+Automatic Analysis
+^^^^^^^^^^^^^^^^^^
+
+YANK looks at a simulation output directory, and makes automatic choices based on best practices and procedures. All
+the steps taken here are outlined above and in order. You can use this ``yank analyze`` facility, even when YANK is
+running.
+
+.. code-block:: bash
+
+  $ yank analyze --store={experiments}
+
+Replace ``{experiments}`` with the output directory for your simulation. The output is printed to standard terminal
+output.
+
+
+.. _visual_analyze:
+
+Visual Analysis
+^^^^^^^^^^^^^^^
+
+YANK can create `Jupyter Notebooks <http://jupyter.org/>`_ to analyze your simulations to help visually see more than
+just walls of text and numbers. These notebooks behave similar to the automatic analysis in that they follow the
+set of data processing and analysis methods, but will render graphic representation of the same data.
+
+.. note::
+
+    Rendering these notebooks requires both the ``juptyer`` and ``matplotlib`` packages and their dependencies. These
```

Comment:
Shouldn't we install these as part of the YANK repo? Otherwise, we're going to run into users who can't analyze their data!